src/traces/box/attributes.js

```diff
@@ -221,12 +221,14 @@ module.exports = {
 
     boxmean: {
         valType: 'enumerated',
-        values: [true, 'sd', false],
+        values: [true, 'sd', '1sigma', '2sigma', '3sigma', '4sigma', '5sigma', '6sigma', false],
```

Comment:
My gut reaction is `boxmean` - which currently is just about adding a little extra information - isn't the right place to describe such a foundational feature as "what does the size of the box mean," especially since `true` is already taken.

I'd propose we make two new attributes:
- `sizemode` - defaults to `quartiles` (current behavior) but you can set `sd` to change to mean ± N sd
- `sdmultiple` - a number rather than an enumeration of a few values. Only available if `sizemode='sd'` or `boxmean='sd'`, defaults to `1` and has minimum `0`, and multiplies sd in either case (box or diamond).

And then if you set `sizemode='sd'`, the `boxmean` attribute is not available.